### DISCUSSION TITLE
Stylelint error since 6.20.1

### DISCUSSION THREAD

**Discussion by jokay** [oxsecurity/megalinter#2466] [Q&A] [answered]
When trying to update to **6.20.1** we get this new error:

![image](https://user-images.githubusercontent.com/18613935/226825322-191359ab-9cfb-4cec-a8fd-f861eff1e588.png)

`.stylelintrc.json`

```json
{
  "extends": [
    "stylelint-config-standard",
    "stylelint-config-recommended-scss",
    "stylelint-config-recommended",
    "stylelint-config-rational-order"
  ],
  "customSyntax": "postcss-scss",
  "overrides": [
    {
      "files": ["**/*.{html,vue}"],
      "customSyntax": "postcss-html"
    }
  ],
  "plugin/rational-order": true,
  "plugins": [
    "stylelint-scss",
    "stylelint-order",
    "stylelint-config-rational-order/plugin"
  ]
}
```

This configuration works using **6.20.0** but no longer using **6.20.1**.

Not sure if it's related to https://github.com/oxsecurity/megalinter/issues/2460?

**Comment by nvuillam** [comment, marked as answer]
@jokay this should be solved by latest MegaLinter v6.21.0, please can you confirm it's ok ? :)

**Reply by jokay** [reply]
Yes, works again, thx a lot :+1:
